src/geometry.cpp

```diff
@@ -259,7 +258,7 @@ bool find_cell_inner(Particle& p, const NeighborList* neighbor_list)
 
 //==============================================================================
 
-bool neighbor_list_find_cell(Particle& p)
+bool neighbor_list_find_cell(Geometron& p, bool verbose)
```

Comment:
Verbose output happens when a specific particle has `trace_` set to true, or when `settings::verbosity` is set to true. Because the `trace_` variable is not carried on the `Geometron` (it is the minimal set of state for geometry operations), this leaves it on the outer calling code to decide verbosity.